### ISSUE TITLE
Cannot add the task to pipeline

### ISSUE THREAD

**Issue by jkonecki** [der-pepe/AzureDevOps-NetFrameworkInstaller#1]
I've installed the task from Marketplace but I'm unable to add it to the actual pipeline.

When adding to the pipeline the task has only a greyed out Installed button:

![image](https://user-images.githubusercontent.com/1775247/58216033-911d6d00-7cf4-11e9-8e78-08c7d582a432.png)

How am I supposed to use the task?

There is a separate .Net Installer task 

![image](https://user-images.githubusercontent.com/1775247/58216098-d17ceb00-7cf4-11e9-8c4c-56eb13131dd1.png)

but when I try to use it I get the error:

##[debug]Evaluating condition for step: 'netinstaller'
##[debug]Evaluating: succeeded()
##[debug]Evaluating succeeded:
##[debug]=> True
##[debug]Result: True
##[section]Starting: netinstaller
==============================================================================
Task         : .NET Installer
Description  : Run .NET Installer
Version      : 1.0.10
Author       : JGomezZ
Help         : Run .NET Installer
==============================================================================
##[error]File not found: 'd:\a\_tasks\netinstaller_bc5b84a6-e2c8-4a5a-bc53-31a79c0de68c\1.0.10\ps_modules\VstsTaskSdk\VstsTaskSdk.psd1'
##[debug]System.IO.FileNotFoundException: File not found: 'd:\a\_tasks\netinstaller_bc5b84a6-e2c8-4a5a-bc53-31a79c0de68c\1.0.10\ps_modules\VstsTaskSdk\VstsTaskSdk.psd1'
File name: 'd:\a\_tasks\netinstaller_bc5b84a6-e2c8-4a5a-bc53-31a79c0de68c\1.0.10\ps_modules\VstsTaskSdk\VstsTaskSdk.psd1'
   at Microsoft.VisualStudio.Services.Agent.Util.ArgUtil.File(String fileName, String name)
   at Microsoft.VisualStudio.Services.Agent.Worker.Handlers.PowerShell3Handler.RunAsync()
   at Microsoft.VisualStudio.Services.Agent.Worker.TaskRunner.RunAsync()
   at Microsoft.VisualStudio.Services.Agent.Worker.StepsRunner.RunStepAsync(IStep step, CancellationToken jobCancellationToken)
##[section]Finishing: netinstaller

**Comment by der-pepe** [comment]
Yes, I'm able to reproduce this issue, I'll fix the task definition and re-publish the extension

**Comment by hoxha-saber** [comment]
Hello, I was wondering if there is any progress on this issue. I am also having this error.

**Comment by g4d** [comment]
Same.

**Comment by g4d** [comment]
@der-pepe is there a work-around in the mean-time?
